CONTRIBUTING.md

```diff
@@ -1,4 +1,9 @@
-# Request for contributions
+# Contributing to OSEM
+Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) and send a PR!
```

Comment:
Not pretty sure if you like what @differentreality wrote or you also want to change it :confused: